sbt-dotty/src/dotty/tools/sbtplugin/DottyPlugin.scala

```diff
@@ -317,12 +324,47 @@ object DottyPlugin extends AutoPlugin {
 
       // Because managedScalaInstance is false, sbt won't add the standard library to our dependencies for us
       libraryDependencies ++= {
-        if (isDotty.value && autoScalaLibrary.value)
-          Seq(scalaOrganization.value %% "dotty-library" % scalaVersion.value)
-        else
+        if (isDotty.value && autoScalaLibrary.value) {
+          val name =
+            if (isDottyJS.value) "dotty-library_sjs1"
+            else "dotty-library"
+          Seq(scalaOrganization.value %% name % scalaVersion.value)
+        } else
           Seq()
       },
 
+      // Patch up some more options if this is Dotty with Scala.js
```

Comment:
Since the build of dotty itself uses sbt-dotty, you should be able to remove the similar patching done in project/Build.scala

Reply:
Oh, I did not know that. That's interesting.